src/styles/mixins/_forms.scss

```diff
@@ -250,6 +249,8 @@
     background: var(--fdInput_Group_Button_Background);
     color: var(--fdInput_Group_Button_Text_Color);
 
+    @include fd-set-border-left(var(--sapButton_BorderWidth) solid transparent);
```

Comment:
What is the point of this anyways? The icon looks just slightly out-of-center in the button and I think this may be the culprit

Reply:
So I think we can have a border width of 0 for non-HC modes